libs/base/ui/graphical/spinner/src/spinner.styles.ts

```diff
@@ -8,7 +8,7 @@ export const styles = css`
     display: flex;
     aspect-ratio: 1 / 1;
     justify-content: center;
-    color: var(--oryx-color-primary-300, inherit);
+    color: var(--oryx-color-primary-9, 'inherit');
```

Comment:
I think you know what I wanna say =)

Reply:
So, here's why I did what I did:
- introduce a function for colors:  convenient way to not o through this heavy refactor in the (near) future when we change our minds on variable names
- export a const for each color, since stylelint does not allow to use the function inline. The const uses the `var(...)` syntax as 99% of the cases needs a straight var
- avoid the const when a fallback is needed, since we cannot use a function